searxng/searxng
# [mod] user_help: support translations

## What does this PR do?

replace "user_help.HELP" by a new function "get_help_for_locale".
This function the help a one locale, and fall back to English if the
translation is not available.

## Why is this change important?

The user documentation is written using a markup language to allow translation.

## How to test this PR locally?

* create a directory in `searx/user_help/` with a local name ( `de` for example).
* start the server, go to `/about?locale=de`, check the English version is displayed
* copy `searx/user_help/en/about.md` to `searx/user_help/de/about.md`
* make some changes
* start the server, go to `/about?locale=de`, check the new version is displayed

## Author's checklist



## Related issues
